packages/tatum/src/nft/tests/celo.spec.ts

```diff
@@ -0,0 +1,334 @@
+      const address = '0x2A42ae2a6346eEbC7FE2b2b7f02158634d5390dc'
+      const nfts = await getNFTsByAddress(Currency.CELO, contractAddress, address)
+      expect(nfts).not.toBeNull()
+      console.log('Get nfts by address: ', nfts)
+    } catch (e) {
+      console.log('Get nfts by address error: ', e.response.data)
+      expect(e).not.toBeDefined()
+    }
+  })
+  it('should get NFT provenance data', async () => {
+    try {
+      const contractAddress = '0x2A42ae2a6346eEbC7FE2b2b7f02158634d5390dc'
+      const tokenId = '1641548895528'
+      const provenanceData = await getNFTProvenanceData(Currency.CELO, contractAddress, tokenId)
+      expect(provenanceData).not.toBeNull()
+      console.log('Get nft provenance data: ', provenanceData)
+    } catch (e) {
+      console.log('Get nft provenance data error: ', e.response.data)
+      expect(e).not.toBeDefined()
+    }
+  })
+  it('should get NFT metadata uri', async () => {
+    try {
+      const contractAddress = '0x2A42ae2a6346eEbC7FE2b2b7f02158634d5390dc'
+      const tokenId = '1641548895528'
+      const metadata = await getNFTMetadataURI(Currency.CELO, contractAddress, tokenId)
+      expect(metadata).not.toBeNull()
+      console.log('Get nft metadata uri: ', metadata)
+    } catch (e) {
+      console.log('Get nft metadata uri error: ', e.response.data)
+      expect(e).not.toBeDefined()
+    }
+  })
+  it('should get NFT image', async () => {
+    try {
+      const contractAddress = '0x2A42ae2a6346eEbC7FE2b2b7f02158634d5390dc'
+      const tokenId = '1641548895528'
+      const image = await getNFTImage(Currency.CELO, contractAddress, tokenId)
+      expect(image).not.toBeNull()
+      console.log('Get nft image: ', image)
+    } catch (e) {
+      console.log('Get nft image error: ', e.response.data)
+      expect(e).not.toBeDefined()
+    }
+  })
+  it('should get NFT royalty', async () => {
+    try {
+      const contractAddress = '0x2A42ae2a6346eEbC7FE2b2b7f02158634d5390dc'
+      const tokenId = '1641548895528'
+      const royalty = await getNFTRoyalty(Currency.CELO, contractAddress, tokenId)
+      expect(royalty).not.toBeNull()
+      console.log('Get nft royalty: ', royalty)
+    } catch (e) {
+      console.log('Get nft royalty error: ', e.response.data)
+      expect(e).not.toBeDefined()
+    }
+  })
+
+  it('should create NFT', async () => {
+    try {
+      const tokenId = new Date().getTime().toString()
+      const body = {
+        to: '0x811dfbff13adfbc3cf653dcc373c03616d3471c9',
+        chain: Currency.CELO,
+        feeCurrency: Currency.CUSD,
+        tokenId: tokenId,
+        url: '',
+        fromPrivateKey: '0d6c13fe5fed644dfa02512d4bffde9453dcb48873afb0b0a4c0cebce160c279',
+        contractAddress: '0x2A42ae2a6346eEbC7FE2b2b7f02158634d5390dc',
+      }
+      const nft = await createNFT(true, body, readFileSync('/Users/marinajakcin/Downloads/pikachu.png'), 'Pokemon', 'Electric type')
```

Comment:
This is coupled to your local environment and will not pass anywhere else because for example my local path is `/Users/martinrucki/...` and I also do not have the image on my computer.

Reply:
previously there was Sam's path unique for his environment... @ruckim is there some way how to read it from neutral source and use "readFileSync" at the same time? If not I would leave it that way now

Reply:
I am thinking about it like this: This function reads file, use some encoding and return some string or Buffer, so I would completely skip the step of reading and provide just some raw data, some string or Buffer if it does not matter what is the content, if content matters, then convert picture to string or Buffer locally and use the result as an argument.

Reply:
Yes I agree this needs to be fixed but I left it just for the sake of speeding things up
The image doesn't matter I think so we can use something random